File: 0485-max-consecutive-ones/0485-max-consecutive-ones.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        maxcount = 0
        count = 0
        for i in range(len(nums)):
            if nums[i] == 0:
                count = 0
            else:
                count += 1
                maxcount = max(maxcount , count)
        return maxcount

File: 0485-max-consecutive-ones/0485-max-consecutive-ones.java
import java.util.*;
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
        int maxcount = 0;
        int count = 0;
        for(int i = 0 ; i < nums.length ; i++){
            if(nums[i] == 0){
                maxcount = Math.max(maxcount , count);
                count = 0;
            }
            else count++;
        }
        maxcount = Math.max(maxcount , count);
        return maxcount;
    }
}